recipe/meta.yaml

```diff
@@ -9,19 +9,20 @@ source:
   sha256: 3744eca3e2284aee3137c8995025bde7ea76d5b78fad6d26e709a8c167c5a61d
 
 build:
-  number: 0
+  number: 1
   script:
     - "{{ PYTHON }} -c 'import os, shutil; shutil.rmtree(os.path.join(\"pythran\", \"xsimd\"))'"
     - "{{ PYTHON }} -c 'with open(\"setup.cfg\", \"w\") as s: s.write(\"[build_py]\\nno_xsimd=True\")'"
+    - "{{ PYTHON }} -c 's = open(\"pythran\\pythran-win32.cfg\", \"r\"); c = s.read(); s.close(); s = open(\"pythran\\pythran-win32.cfg\", \"w\"); s.write(c.replace(\"CC=\", \"CC=clang\").replace(\"CXX=\",\"CXX=clang++\")); s.close()'"  # [win]
     - "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: true  # [win and py27]
   entry_points:
     - pythran = pythran.run:run
     - pythran-config = pythran.config:run
 
 requirements:
   build:
-    - {{ compiler('cxx') }}
+    - {{ compiler('cxx') }}  # [not win]
+    - clang  # [win]
```

Comment:
That's strange. Why are you not happy with the default compiler?

Reply:
Visual compiler fails to compile many of pythran-generated compiler, I'm trying to use clang-cl instead (but currently fails :-/)